.gitignore

```diff
@@ -6,6 +6,9 @@
 .idea/
 exchanger
 
+# Logs of the binary
+cmd/logs/*
+
 # Test binary, build with `go test -c`
 *.test
 
```

Comment:
Add exchanger.config to gitignore and add exchanger.config-example to cmd folder